internal/connectors/plugins/public/wise/payouts.go

```diff
@@ -0,0 +1,87 @@
+package wise
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/formancehq/go-libs/v2/pointer"
+	"github.com/formancehq/payments/internal/connectors/plugins/currency"
+	"github.com/formancehq/payments/internal/connectors/plugins/public/wise/client"
+	"github.com/formancehq/payments/internal/models"
+)
+
+func (p *Plugin) createPayout(ctx context.Context, pi models.PSPPaymentInitiation) (models.PSPPayment, error) {
+	if err := p.validateTransferPayoutRequest(pi); err != nil {
+		return models.PSPPayment{}, err
+	}
+
+	sourceProfileID := pi.SourceAccount.Metadata["profile_id"]
+	destinationProfileID, _ := strconv.ParseUint(pi.DestinationAccount.Metadata["profile_id"], 10, 64)
+
+	curr, precision, err := currency.GetCurrencyAndPrecisionFromAsset(supportedCurrenciesWithDecimal, pi.Asset)
+	if err != nil {
+		return models.PSPPayment{}, fmt.Errorf("failed to get currency and precision from asset: %w: %w", err, models.ErrInvalidRequest)
+	}
+
+	amount, err := currency.GetStringAmountFromBigIntWithPrecision(pi.Amount, precision)
+	if err != nil {
+		return models.PSPPayment{}, fmt.Errorf("failed to convert amount to string: %w: %w", err, models.ErrInvalidRequest)
+	}
+
+	quote, err := p.client.CreateQuote(ctx, sourceProfileID, curr, json.Number(amount))
+	if err != nil {
+		return models.PSPPayment{}, err
+	}
+
+	resp, err := p.client.CreatePayout(ctx, quote, destinationProfileID, pi.Reference)
+	if err != nil {
+		return models.PSPPayment{}, err
+	}
+
+	payment, err := fromPayoutToPayment(*resp)
+	if err != nil {
+		return models.PSPPayment{}, err
+	}
+
+	return *payment, nil
```

Comment:
`fromPayoutToPayment` sometimes returns `nil, nil` so we can't dereference payment here without the risk of a panic.

What is the behaviour we expect here? Just to skip initiating a payment silently if the precision of the currency can't be determined? (seems more like a case we should reject...)